kernel/api/src/main/java/org/sakaiproject/hikaricp/jdbc/pool/SakaiBasicDataSource.java

```diff
@@ -175,7 +175,7 @@ protected void init() throws SQLException
                 } catch (ClassNotFoundException | InstantiationException | IllegalAccessException e) {
                     // TODO Auto-generated catch block
                     String message = "driverClass not specified, might not be able to load the driver'";
-                    M_log.warn(message, e);
+                    M_log.error(message, e);
```

Comment:
Well on the hikaricp documentation it said "When using this property (jdbcUrl) with "old" drivers, you may also need to set the driverClassName property, but try it first without."

I believe I had this in there because there are 2 ways to load drivers. It might be *possible* that it would still work if you specified an incorrect driverClassName name, but it would be REALLY obvious if it didn't work far beyond this warning. So this really might be more of an info than anything saying that you really didn't need to specify the driverClassName property in the first place.